universal-application-tool-0.0.1/app/services/applicant/ApplicantQuestion.java

```diff
@@ -127,6 +127,9 @@ public PresentsErrors errorsPresenter() {
 
   public class AddressQuestion implements PresentsErrors {
```

Comment:
I would *really* like to refactor `ApplicantQuestion` to move these subclasses into their own files so that we can unit test all their validation logic. I'll look for the issue for that and add a TODO.

Reply:
This is resolved now that #656 is in!